### PR TITLE
cratification: move the bytes command to nu-cmd-extra

### PR DESCRIPTION
**Pull request by stormasm**
now that #9455 has landed we can move the bytes command to nu-cmd-extra

in concert with

moving nu_command::util to nu-cmd-base

### CODE REVIEW

File: crates/nu-cmd-extra/src/extra/bytes/mod.rs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+pub(crate) mod build_;
+pub(crate) mod bytes_;
+pub(crate) mod collect;
+pub(crate) mod ends_with;
+pub(crate) mod index_of;
+pub(crate) mod length;
+pub(crate) mod remove;
+pub(crate) mod replace;
+pub(crate) mod reverse;
+pub(crate) mod starts_with;
```
Comment (sholderbach) [inline crates/nu-cmd-extra/src/extra/bytes/mod.rs:12]: What is the need to make this blanket visible? There have been recently a number of compiler warnings due to potential namespace conflicts due to many similar named symbols imported via `use module::*` that arise from symbols being public.

Comment (stormasm) [reply crates/nu-cmd-extra/src/extra/bytes/mod.rs]: @sholderbach thanks great catch ! I made your requested changes....

**Comment by stormasm** [comment]
@amtoine I know you are moving this weekend and @sholderbach kindly reviewed the PR so I am going to go ahead and land this...

I am going to circle back on to the bits commands after this lands and make the same changes @sholderbach suggested for this PR on the visibility issue of not needing the blanket pub(crate) which I corrected in this PR thanks to his suggestion.

**Review by sholderbach — APPROVED** [review]
Should be good but marked as a breaking change as it removes commands from the default build

**Comment by amtoine** [comment]
thanks @stormasm and @sholderbach for backing me up on that one, really appreciate :elephant: :heart:

**Comment by amtoine** [comment]
and it looks all good and working as expected to me, nice job @stormasm :blush:

**Pull request merged**